Palace/Reader2/Bookmarks/TPPBookmarkSpec.swift

```diff
@@ -171,7 +185,11 @@ struct TPPBookmarkSpec {
        bookID: String,
        selectorValue: String) {
     self.id = Id(value: id)
-    self.body = Body(time: time.rfc3339String(), device: device)
+    
+    let dict = try? JSONSerialization.jsonObject(with: selectorValue.data(using: .utf8)!, options: []) as? [String: Any]
+    let title = (dict!!["title"] as? String) ?? ""
```

Comment:
Maybe we can avoid double force-unwrapping here? Don't know if `selectorValue` can be an empty string `""`, but the app will crash in this case.

Reply:
Nice catch, I meant to go back and fix this, it was for debugging purposes only. Will update